.circleci/config.yml

```diff
@@ -175,6 +194,37 @@ workflows:
           requires:
             - run_specs
             - linters
+      - deploy_uat:
+          context: laa-estimate-eligibility-uat
+          requires:
+            - build_and_push_to_ecr
+
+  merge_pr_to_main:
+    jobs:
+      - install_dependencies:
+          filters:
+            branches:
+              only: main
+      - linters:
+          filters:
+            branches:
+              only: main
+          requires:
+            - install_dependencies
+      - run_specs:
+          filters:
+            branches:
+              only: main
```

Comment:
As in `linters` job. Keep conversation there, but if that conversation results in a change it will likely be needed here too.